.github/workflows/ci.yml

```diff
@@ -0,0 +1,43 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - '*'
+  pull_request:
+    branches:
+      - '*'
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        ruby: [ '2.6.3' ]
+      fail-fast: true
+    runs-on: ubuntu-latest
+    env:
+      CUSTOM_RUBY_VERSION: ${{ matrix.ruby }}
```

Comment:
What's this for? 🙋‍♂

Reply:
To work around GitHub Actions not supported 2.6.5.

Reply:
To overide the default Ruby version in the Gemfile (2.6.5); else it would abort.